### PR TITLE
added enabled prop with default true, can disable component from parent

### PR DESCRIPTION
**Pull request by jgrisafe**
This allows the parent to set an enabled prop, so that if we want to only have the scroll lock active at specific times we can do so. One example is on a text field, if the user is not focused on that field, you may not want the scroll lock to be active or it could disrupt the flow of scrolling down the page.

### CODE REVIEW

File: src/ScrollLock.jsx [hunk 1]
```diff
@@ -2,7 +2,18 @@ import React, { Component } from 'react';
 
 class ScrollLock extends Component {
   componentDidMount() {
-    this.scrollingElement.addEventListener('wheel', this.onScrollHandler, false);
+    const { enabled } = this.props
+    if (enabled) {
```
Comment (orteth01) [inline src/ScrollLock.jsx]: do you mind just doing this instead of the destructuring?

```
if (this.props.enabled) {
```

**Review by orteth01 — CHANGES_REQUESTED** [review]
just requesting the one change.

**Comment by orteth01** [comment]
@jgrisafe thanks for taking this on! sorry it took so long for me to get to this!

**Comment by jgrisafe** [comment]
@orteth01 awesome, done!

**Review by orteth01 — APPROVED** [review]

**Comment by orteth01** [comment]
@jgrisafe i'll bump minor version when I get home from work. thanks again!

**Pull request merged**